### ISSUE TITLE
[Button][Firefox][HighContrast] Button not following high contrast standards

### ISSUE THREAD

**Issue by ankitk94** [OfficeDev/office-ui-fabric-react#3570]
### Bug Report

- __Browser and OS versions__: Firefox

#### Describe the issue:

Button not following high contrast standards on firefox (working as expected on edge)

#### Actual behavior (on Firefox):
HC black:
![image](https://user-images.githubusercontent.com/6390909/33934318-92463532-e01e-11e7-992d-e502e298083b.png)

HC white:
![image](https://user-images.githubusercontent.com/6390909/33934399-cc5f6824-e01e-11e7-87b2-af484e6a91d5.png)

#### Expected behavior (on Edge):
HC black:
![image](https://user-images.githubusercontent.com/6390909/33934364-aefb743a-e01e-11e7-9391-0624f8d7ca2c.png)

HC white:
![image](https://user-images.githubusercontent.com/6390909/33934449-ee4f8afe-e01e-11e7-8fa6-56112dd1c5e7.png)

### If applicable, please provide a codepen repro:

Repro on official fabric website:
https://developer.microsoft.com/en-us/fabric#/components/button

**Comment by betrue-final-final** [comment]
@kristoferbrown, has your work been checked in? Your PR may fix it if I'm correct?

**Comment by kristoferbrown** [comment]
That PR specifically addressed high contrast issues in Edge/IE. Firefox is missing a lot of support for methods to control high contrast mode so solving contrast issues there is much more difficult. Keep this issue open so we can address this in the future.

**Comment by michaelangotti** [comment]
@ankitk94  We have concluded that we cannot reasonably support Firefox HighContrast. We do have PR's open to address this Edge and IE. Will close this for now, but will revisit in the future.
